File: c/chapitre_3/exo_31.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc , char **argv)
{
    int somme ;

    if (argc != 3 )
    {
        perror("Vous devez inserez 2 arguments\n");
        exit(-1);
    }
    somme = atoi(argv[1]) + atoi(argv[2]) ;
    printf("Somme : %d \n",  somme ) ;

    return 0 ;
}

File: python/chapitre_3/exo31.py
import sys

def main():
    if len(sys.argv) != 3:
        print("Vous devez inserez 2 arguments")
        sys.exit(-1)

    somme = int(sys.argv[1]) + int(sys.argv[2])
    print("Somme :", somme)

if __name__ == "__main__":
    main()